Automattic/wp-calypso
# Checklist: Hide banner from too old sites.

The onboarding checklist may not work properly for the sites created before the feature was launched due to lack of `onboarding_checklist_initialized` option that is generated when a site is created. This PR will check if the site is eligible for the checklist and hide the banner if not.

_Note_: You need to apply D17548-code to your sandbox first.

## Test plan

1. Create a new site.
2. See if the checklist banner shows up on the stats page. You should be able to see it.
3. Repeat the above step on the site created before Feb 1, 2018. But this time, the banner should not appear on the page.
